### DISCUSSION TITLE
DEM-Structure Interaction

### DISCUSSION THREAD

**Discussion by mnabideltares** [KratosMultiphysics/Kratos#11306] [General]
We are interested on DEM-Structure interaction. We would like to simulate pipe/beam inside small stones, and simulate the effect of vibration of pipes on the soil.

Is there anything available as example (or similar)? If primilinary results/movie is also ok.

Is there any documentation available explaining how to visualize DEM-FEM in GID?

Thank you.

**Comment by ChengshunShang1996** [comment]
Hello, it is very interesting topic. I am not sure whether you have solved you questions. As I know, you may have a look on the CoSimulationApplication to find some DEM-FEM couping wrapper and cases. I am woring on geomechanics (mainly rock) using Kratos DEM and have carried out some DEM-FEM couping simulations. If you need more information, we can discuss a bit more.
